bundle/Controller/UserController.php

```diff
@@ -155,20 +155,21 @@ public function editAction(
         ?string $language = null,
         Request $request
     ) {
+        $languageCode = $language;
```

Comment:
You can simply rename `$language` parameter to `$languageCode`, it won't break BC. 

```suggestion
```